components/integrations/installed_incoming_webhook.tsx

```diff
@@ -166,7 +180,7 @@ export default class InstalledIncomingWebhook extends React.PureComponent {
                                 id='installed_integrations.creation'
                                 defaultMessage='Created by {creator} on {createAt, date, full}'
                                 values={{
-                                    creator: this.props.creator.username,
+                                    creator: this.props?.creator?.username,
```

Comment:
creator was originally required, this can change to its prior implementation after making it required in the Props type again